glamorous/webpack.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        exclude: /(node_modules)/,
+        loader: 'babel',
+        query: {
+          presets: ['es2015', 'react']
```

Comment:
Oh, actually, could we use `babel-preset-env` instead? The babel team is really trying to discourage the use of pretty much every other ES preset.